css/style.css

```diff
@@ -0,0 +1,735 @@
+    color: #FFFFFF;
+    width: 80ch;
+}
+
+.skills-page {
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    width: 100%;
+    height: auto;
+    padding: 20px 0;
+    background-image: url("../images/skills-page/skills-background.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    position: relative;
+}
+
+.skills-page::before {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(198, 217, 223, 0.7);
+    z-index: 1;
+}
+
+.soft-skills,
+.hard-skills {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    justify-content: flex-start;
+    align-items: center;
+    width: 490px;
+    height: 550px;
+    background-color: rgba(92, 88, 164, 0.06);
+    box-shadow: 0px 0px 10px rgb(69, 111, 130);
+    padding: 20px 20px;
+    overflow-y: auto;
+    z-index: 2;
+    scrollbar-color: #5F8FA7 rgba(95, 143, 167, 0.20);
+}
+
+.soft-skills h2,
+.hard-skills h2 {
+    font-family: "Copse", serif;
+    font-weight: 400;
+    font-style: normal;
+    font-size: 25px;
+    color: #588CA4;
+}
+
+.soft-skills p {
+    font-family: "Delius", cursive;
+    font-weight: 400;
+    font-style: normal;
+    font-size: 20px;
+    background-color: rgba(135, 171, 188, 0.06);
+    border: 1px solid #588CA4;
+    width: 90%;
+    padding: 20px;
+    color: #588CA4;
+}
+
+.hard-skills-inner-container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    align-items: center;
+    gap: 20px 60px;
+}
+
+.hard-skills-inner-container p {
+    font-family: "Delius", cursive;
+    font-weight: 400;
+    font-style: normal;
+    font-size: 22px;
+    height: 60px;
+    width: 60px;
+    background-color: rgba(88, 140, 164, 0.06);
+    border-radius: 50%;
+    border: 1px solid #588CA4;
+    padding: 48px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: #588CA4;
+}
+
+.educational-background-page {
+    width: 100%;
+    min-height: 600px;
+    height: auto;
+    max-height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-image: url("../images/education-page/education-background.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+}
+
+.timeline {
+    position: relative;
+    max-width: 1200px;
+    margin: 0 auto;
+}
+
+.timeline::after {
+    content: '';
+    position: absolute;
+    width: 6px;
+    background-color: #D5E3E7;
+    top: 0;
+    bottom: 0;
+    left: 50%;
+    margin-left: -3px;
+}
+
+.container {
+    padding: 10px 40px;
+    position: relative;
+    background-color: inherit;
+    width: 50%;
+}
+
+.container::after {
+    content: '';
+    position: absolute;
+    width: 25px;
+    height: 25px;
+    right: -17px;
+    background-color: #588CA4;
+    border: 4px solid #D5E3E7;
+    top: 15px;
+    border-radius: 50%;
+    z-index: 1;
+}
+
+.left {
```

Comment:
Using "left" and "right" as class names is not clear. Let's establish a better naming convention for the classes and the same for other classes, for example:

- **Block**: A standalone component (e.g., navbar, button).
- **Element**: A part of the block (e.g., navbar__item, button__icon).
- **Modifier**: A variation or state of the block/element (e.g., button--primary, button__icon--large).

Reply:
I applied this naming convention to the entire code after refactoring, but some names have become long and complex, so I hope they're still understandable and match the convention.